### ISSUE TITLE
Not play in chrome 66

### ISSUE THREAD

**Issue by frank780117** [leoasis/react-sound#67]
hi, sometimes not play sound, can you add mute or other property then I can use.

https://stackoverflow.com/questions/49930680/how-to-handle-uncaught-in-promise-domexception-play-failed-because-the-use

**Comment by glodfinch** [comment]
I also have this issue with Chrome. Strangely sometimes the sounds will play and sometimes they wont.
`Uncaught (in promise) DOMException: play() failed because the user didn't interact with the document first. https://goo.gl/xX8pDD`

**Comment by alaingoldman** [comment]
Same issue as above ^

**Comment by pallavi1993** [comment]
It is because of the chrome's browser setting.

**Comment by justintemps** [comment]
Indeed. It's because of this: https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

You need to listen for a user interaction before you set the `playStatus` to `Sound.status.PLAYING`

**Comment by andykotov** [comment]
Please help to resolve this issue, I have the same warning

`soundmanager2.js:1989 Uncaught (in promise) DOMException: play() failed because the user didn't interact with the document first. https://goo.gl/xX8pDD`

**Comment by derakhshanfar** [comment]
any update here guys?

**Comment by glodfinch** [comment]
This turned out to be exactly what the error says - the user needs to interact with the page before you can autoplay sounds. I now have a small banner at the top of the page asking for autoplay permission, with a button for the user to click. Works great.
